code/modules/mob/living/carbon/xenomorph/castes/Hivelord.dm

```diff
@@ -115,3 +115,10 @@
 	toggle_resin_walker()
 	to_chat(bound_xeno, SPAN_WARNING("You feel dizzy as the world slows down."))
 	bound_xeno.recalculate_move_delay = TRUE
+
+/// This check mainly exists because of the new resin node ability for resin whisperer.
+/mob/living/carbon/xenomorph/hivelord/proc/on_weeds()
+    var/turf/T = get_turf(src)
+    if(locate(/obj/effect/alien/weeds) in T)
+        return TRUE
+    return FALSE
```

Comment:
this is using space indentation which linters hate

Reply:
Thank you!